views/account-edit.pug

```diff
@@ -1,41 +1,39 @@
-.user-info.text-center
-  h4 #{user.id}
-  p #{user.fname} #{user.lname}
-  p #{user.email}
-  p #{user.points} Points
-  p Created on #{cdate}
+.row
+  .col.s12
+    .card.center
+      .card-content
+        h5.card-title Account Edit
+        h6 Modify and update #{user.id}'s information.
+      .card-action
+        form.form-edit#account-edit-form
+          .form-group.row
+            label.col-sm-2.col-form-label(for='userid') Username
+            .col-sm-10
+              input.form-control#userid(type='text' name='id' value=user.id autofocus)
 
-  p#account-edit-result
+          .form-group
+            .row
+              label.col-sm-2.col-form-label(for='fname') First Name
+              .col-sm-10
+                input.form-control#fname(type='text' name='fname' value=user.fname)
+            .row
+              label.col-sm-2.col-form-label(for='lname') Last Name
+              .col-sm-10
+                input.form-control#lname(type='text' name='lname' value=user.lname)
 
-form.form-edit#account-edit-form
-  .form-group.row
-    label.col-sm-2.col-form-label(for='userid') Username
-    .col-sm-10
-      input.form-control#userid(type='text' name='id' placeholder='New username' autofocus)
+          .form-group.row
+            label.col-sm-2.col-form-label(for='email') Email
+            .col-sm-10
+              input.form-control#email(type='email' name='email' value=user.email)
 
-  .form-group
-    .row
-      label.col-sm-2.col-form-label(for='fname') First Name
-      .col-sm-10
-        input.form-control#fname(type='text' name='fname' placeholder='New first name')
-    .row
-      label.col-sm-2.col-form-label(for='lname') Last Name
-      .col-sm-10
-        input.form-control#lname(type='text' name='lname' placeholder='New last name')
+          .form-group.row
+            label.col-sm-2.col-form-label(for='points') Points
+            .col-sm-10
+              input.form-control#email(type='number' name='points' value=user.points)
 
-  .form-group.row
-    label.col-sm-2.col-form-label(for='email') Email
-    .col-sm-10
-      input.form-control#email(type='email' name='email' placeholder='New email')
+          .form-group.row
```

Comment:
Indentation is off here, 4 spaces instead of 2. I thought Pug was really picky about this?

Reply:
fixed it